app/models/dialog.rb

```diff
@@ -1,8 +1,9 @@
 class Dialog < ApplicationRecord
-  DIALOG_DIR = Rails.root.join("product/dialogs/service_dialogs")
+  DIALOG_DIR_CORE   = 'product/dialogs/service_dialogs'.freeze
+  DIALOG_DIR_PLUGIN = 'content/service_dialogs'.freeze
```

Comment:
IWe can drop the `DIALOG_DIR_CORE` logic once all dialogs have been moved to provider pugins or manageiq-content